test/Mvc.Api.Analyzers.Test/TestFiles/AddResponseTypeAttributeCodeFixProviderIntegrationTest/CodeFixAddsMissingStatusCodes.Output.cs

```diff
@@ -5,8 +5,8 @@
     public class CodeFixAddsMissingStatusCodes : ControllerBase
     {
         [ProducesResponseType(404)]
-        [ProducesResponseType(200)]
-        [ProducesResponseType(400)]
+        [ProducesResponseType(Http.StatusCodes.Status200OK)]
```

Comment:
Would it make sense to try adding this as a `using`? In the most common case, controllers have a reference to `Microsoft.AspNetCore.Mvc` whereas `Microsoft.AspNetCore.Http` is a rarity.

Reply:
Yeah, the thought did cross my mind. Your call 😊

Reply:
Let's go with the `using`. The results would look a bit cleaner.